web/src/edit/NeighbourhoodMode.svelte

```diff
@@ -88,26 +118,70 @@
   }
 
   function onClickLine(f: Feature, pt: LngLat) {
-    if (action == "filter") {
+    if (action.kind == "filter") {
       $backend!.addModalFilter(pt, $filterType);
       $mutationCounter++;
-    } else if (action == "oneway") {
+    } else if (action.kind == "oneway") {
       $backend!.toggleTravelFlow(f.properties!.road);
       $mutationCounter++;
+    } else if (action.kind == "turn_restriction") {
+      action.from_road_id = f.properties!.road;
+      action.from_road_name = f.properties!.name || "unnamed road";
+      action.possible_targets = $backend!.getTurnRestrictionTargets(
+        f.properties!.road,
+      );
+      // TODO The blue hover state gets stuck
```

Comment:
Click a road to start making a TR, then click the map to deselect. Blue hover stays for reasons unknown. If you hover over any road, then it resets. I'll fix later, unless you spot the problem easily in review